documentation/query.adoc

```diff
@@ -57,111 +57,173 @@ For example, try click on the `Product` node label and you'll see this query pop
+Besides the sidebar with the overview, a quick way to show the current graph schema is to `CALL` this procedure.
 
 [source, cypher]
 ----
-CALL db.schema.visualization
+CALL db.schema.visualization()
 ----
 
-Once you have an idea of the structure of the data, it's time to use Cypher to answer questions and solve problems.
-
 Let's take a look at the different companies supplying Northwind with products and see which supplier provides products from which categories:
 
 [source, cypher]
 ----
 MATCH (s:Supplier)-->(:Product)-->(c:Category)
-RETURN s.companyName as Company, collect(distinct c.categoryName) as Categories
+RETURN s.companyName as company, collect(distinct c.categoryName) as categories
 ----
 
-This query finds _suppliers_ that _supply_ a _product_ which is part of a _category_ and returns the company names and the categories they supply products from, in a table.
-In cases where the same supplier supplies products from more than one category, the categories are collected and listed together, as you can see in the _Categories_ column.
-Note that even though you specified the _Product_ node in the `MATCH` part of the query, no information about the actualy product is returned since this is not included in the `RETURN` part.
+This query finds _suppliers_ that _supply_ a _product_ which is _part of_ a _category_ and returns the company names and the categories they supply products from, in a table.
+In cases where the same supplier supplies products from more than one category, the distinct category names are collected into a list, as you can see in the _categories_ column.
+
+Even though you specified the _Product_ node in the `MATCH` part of the query, no information about the actual product is returned since this is not included in the `RETURN` part.
 
 Maybe you are interested only in suppliers of a certain category, say _Condiments_:
 
 [source, cypher]
 ----
-MATCH (c:Category {categoryName:"Condiments"})<--(:Product)<--(s:Supplier)
-RETURN DISTINCT s.companyName as CondimentsSuppliers
+MATCH (s:Supplier)-->(:Product)-->(c:Category)
+WHERE c.categoryName = 'Condiments'
+RETURN DISTINCT s.companyName as condimentsSuppliers
 ----
 
-This query finds a category with the name _Condiments_ (one of the properties on the _Category_ node, if you recall the data model) and links that to a _Product_ and the product to a _Supplier_ that supplies it.
-The query uses _variables_, `c` and `s`, for the category and the supplier, respectively, but _not_ for the product.
-That's because the product happens to be on the path between the two types of nodes you are interested in, just like in the previous example.
-Note that since you are only interested in that a relationship exists between these nodes and not in _what type_ of relationship, you don't have to specify any relationship type in the query.
+Here we're not interested in the `Product` nodes, nor the exact types of the relationships between the nodes, so we don't have to specify them.
```

Comment:
```suggestion
Since you are not interested in the `Product` nodes, nor the exact types of the relationships between the nodes here, you don't have to specify them.
```